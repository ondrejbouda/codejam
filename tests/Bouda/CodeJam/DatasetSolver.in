3
testCase1
testCase2
testCase3