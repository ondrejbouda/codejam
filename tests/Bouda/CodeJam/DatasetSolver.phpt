<?php

namespace BoudaTests;

use Bouda;
use Tester;
use Tester\Assert;

require_once __DIR__ . '/../../bootstrap.php';

$inputFile = 'DatasetSolver.in';
$outputFile = Tester\FileMock::create('');
$outputFileExpected = 'DatasetSolver.out.expected';

$solver = new Bouda\CodeJam\DatasetSolver($inputFile, new Bouda\CodeJam\CaseSolverNaive);

$solver->solveToFile($outputFile);

Assert::matchFile($outputFileExpected, file_get_contents($outputFile));
