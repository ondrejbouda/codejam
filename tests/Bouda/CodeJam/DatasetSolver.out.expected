Case #1: testCase1
Case #2: testCase2
Case #3: testCase3
